caravel/templates/caravel/basic.html

```diff
@@ -21,7 +21,7 @@
 
   <body>
     {% block navbar %}
-      {% if not viz or viz.request.args.get("standalone") != "true" %}
+      {% if not dash_standalone and (not viz or viz.request.args.get("standalone") != "true") %}
```

Comment:
Do we hide the navbar in any other cases besides explore-standalone? I think in views.py we render standalone.html when standalone=true in the url. Seems like {% if not viz or viz.request.args.get("standalone") != "true" %} will always be true, since standalone.html doesn't extend basic.html